---
title: "Investigate differences of highly expressed genes in each proteome subtype"
output:
  html_document:
    df_print: paged
---

# Introduction

## Background

Lung cancer is a type of cancer that begins in the lungs and has a high mortality rate among cancers. These lung cancers are usually known to be caused by smoking. However, it has been observed a high percentage of the lung cancer patients in East Asia regions are non-smokers.

## [Portfolio] 'Visualize this' - Topic

Investigate differences of highly expressed genes in each proteome subtype
-visualize its expression according to stage and gender.
-Visualize its differences of RNA and proteomic expression according to stage and gender.  



```{r warning = FALSE, message = FALSE}
library(tidyverse)
library(readxl)
library(ggridges)
library(cowplot)
#install.packages('rio')
NSLC_1A = rio::import('https://ars.els-cdn.com/content/image/1-s2.0-S0092867420307431-mmc1.xlsx', sheet = 2)
NSLC_1D = rio::import('https://ars.els-cdn.com/content/image/1-s2.0-S0092867420307431-mmc1.xlsx', sheet = 5)
NSLC_1E = rio::import('https://ars.els-cdn.com/content/image/1-s2.0-S0092867420307431-mmc1.xlsx', sheet = 6, na = "NA")
NSLC_6A = rio::import('https://ars.els-cdn.com/content/image/1-s2.0-S0092867420307431-mmc6.xlsx', sheet = 2)
```

First of all, I would like to check the NSLC_6A among the downloded data. 
 
I tried to compare the gene expression by stage per genders by selecting the five genes which shows the high average values from the patient group in the each proteome subtype.  

```{r warning = FALSE, message = FALSE}
NSLC_6A <- t(NSLC_6A)
NSLC_6A <- as.data.frame(NSLC_6A)
colnames(NSLC_6A)<-NSLC_6A[1,]
NSLC_6A<-NSLC_6A[-c(1,3,4,5),-c(9:15)]
NSLC_6A<-NSLC_6A[,-c(10:72)]
```

This process is to obtain the average expression amount of each gene (average of each column), and will be obtained for each proteome subtype.

```{r warning = FALSE, message = FALSE}
NSLC_6A_pro <- NSLC_6A[-c(1),-c(2:9)]
```

Here 5 genes with the highest expression on average were selected from each proteome subtype.

```{r warning = FALSE, message = FALSE}
NSLC_6A_pro1 <- NSLC_6A_pro %>%
  filter(`proteome_k3 (out of k4)` == 1)

numericvars1 <- NULL

for(i in 2:ncol(NSLC_6A_pro1)) {       
  NSLC_6A_pro1[,i] <- as.numeric(NSLC_6A_pro1[,i])
  numericvars1[i] <- mean(NSLC_6A_pro1[,i], na.rm=TRUE)
}
numericvars1 %>%
  as.data.frame() %>%
  order(decreasing = TRUE) %>%
  head(5)

NSLC_6A_pro2 <- NSLC_6A_pro %>%
  filter(`proteome_k3 (out of k4)` == 2) 

numericvars2 <- NULL

for(i in 2:ncol(NSLC_6A_pro2)) {       
  NSLC_6A_pro2[,i] <- as.numeric(NSLC_6A_pro2[,i])
  numericvars2[i] <- mean(NSLC_6A_pro2[,i], na.rm=TRUE)
}
numericvars2 %>%
  as.data.frame() %>%
  order(decreasing = TRUE) %>%
  head(5)

NSLC_6A_pro3 <- NSLC_6A_pro %>%
  filter(`proteome_k3 (out of k4)` == 3) 

numericvars3 <- NULL

for(i in 2:ncol(NSLC_6A_pro3)) {       
  NSLC_6A_pro3[,i] <- as.numeric(NSLC_6A_pro3[,i])
  numericvars3[i] <- mean(NSLC_6A_pro3[,i], na.rm=TRUE)
}
numericvars3 %>%
  as.data.frame() %>%
  order(decreasing = TRUE) %>%
  head(5)
```
These are NSLC_6A_pro(1,2,3) indices of genes that I want to study. 
It is to be noted that the difference between index of same column of NSLC_6A_pro(1,2,3) and NSLC_6A is 8. 

Now, within each 'proteome subtype' group of those genes, the amount of gene expression by stage according to gender will be shown through plot.

Making plot.
```{r warning = FALSE, message = FALSE}
graph1 <- NSLC_6A %>%
  filter(`proteome_k3 (out of k4)` == 1) %>%
  mutate(OverallStaging =  factor(OverallStaging)) %>%
  mutate(Q96HF1= as.numeric(Q96HF1)) %>%
  ggplot(aes(x = OverallStaging, y = Q96HF1, color = Gender)) +
  geom_violin(scale = "width", alpha = 0.3, adjust = 0.5, aes(fill = Gender)) +
  geom_boxplot(position = position_dodge(0.9), width = 0.2) +
  geom_point(position = position_jitterdodge(jitter.width = 0.3, jitter.height = 0, seed = 12), alpha = 0.9) +
  stat_summary(fun="mean", geom="point",shape = 23, size=5, aes(fill = Gender)) +
  labs(title = "Proteome subtype 1 - Gene SFRP2",
       x = "Stage",
       y = "Gene SFRP2") +
  theme_minimal_grid(12)
```


```{r warning = FALSE, message = FALSE}
graph2 <- NSLC_6A %>%
  filter(`proteome_k3 (out of k4)` == 1) %>%
  mutate(OverallStaging =  factor(OverallStaging)) %>%
  mutate(O75339= as.numeric(O75339)) %>%
  ggplot(aes(x = OverallStaging, y = O75339, color = Gender)) +
  geom_violin(scale = "width", alpha = 0.3, adjust = 0.5, aes(fill = Gender)) +
  geom_boxplot(position = position_dodge(0.9), width = 0.2) +
  geom_point(position = position_jitterdodge(jitter.width = 0.3, jitter.height = 0, seed = 12), alpha = 0.9) +
  stat_summary(fun="mean", geom="point",shape = 23, size=5, aes(fill = Gender)) +
  labs(title = "Proteome subtype 1 - Gene CILP",
       x = "Stage",
       y = "Gene CILP") +
  theme_minimal_grid(12)
```


```{r warning = FALSE, message = FALSE}
graph3 <- NSLC_6A %>%
  filter(`proteome_k3 (out of k4)` == 1) %>%
  mutate(OverallStaging =  factor(OverallStaging)) %>%
  mutate(Q96N03= as.numeric(Q96N03)) %>%
  ggplot(aes(x = OverallStaging, y = Q96N03, color = Gender)) +
  geom_violin(scale = "width", alpha = 0.3, adjust = 0.5, aes(fill = Gender)) +
  geom_boxplot(position = position_dodge(0.9), width = 0.2) +
  geom_point(position = position_jitterdodge(jitter.width = 0.3, jitter.height = 0, seed = 12), alpha = 0.9) +
  stat_summary(fun="mean", geom="point",shape = 23, size=5, aes(fill = Gender)) +
  labs(title = "Proteome subtype 1 - Gene VSTM2L",
       x = "Stage",
       y = "Gene VSTM2L") +
  theme_minimal_grid(12) 
```



```{r warning = FALSE, message = FALSE}
graph4 <- NSLC_6A %>%
  filter(`proteome_k3 (out of k4)` == 1) %>%
  mutate(OverallStaging =  factor(OverallStaging)) %>%
  mutate(P35442= as.numeric(P35442)) %>%
  ggplot(aes(x = OverallStaging, y = P35442, color = Gender)) +
  geom_violin(scale = "width", alpha = 0.3, adjust = 0.5, aes(fill = Gender)) +
  geom_boxplot(position = position_dodge(0.9), width = 0.2) +
  geom_point(position = position_jitterdodge(jitter.width = 0.3, jitter.height = 0, seed = 12), alpha = 0.9) +
  stat_summary(fun="mean", geom="point",shape = 23, size=5, aes(fill = Gender)) +
  labs(title = "Proteome subtype 1 - Gene THBS2",
       x = "Stage",
       y = "Gene THBS2") +
  theme_minimal_grid(12) 
```


```{r warning = FALSE, message = FALSE}
graph5 <- NSLC_6A %>%
  filter(`proteome_k3 (out of k4)` == 1) %>%
  mutate(OverallStaging =  factor(OverallStaging)) %>%
  mutate(P12107= as.numeric(P12107)) %>%
  ggplot(aes(x = OverallStaging, y = P12107, color = Gender)) +
  geom_violin(scale = "width", alpha = 0.3, adjust = 0.5, aes(fill = Gender)) +
  geom_boxplot(position = position_dodge(0.9), width = 0.2) +
  geom_point(position = position_jitterdodge(jitter.width = 0.3, jitter.height = 0, seed = 12), alpha = 0.9) +
  stat_summary(fun="mean", geom="point",shape = 23, size=5, aes(fill = Gender)) +
  labs(title = "Proteome subtype 1 - Gene COL11A1",
       x = "Stage",
       y = "Gene COL11A1") +
  theme_minimal_grid(12) 
```


```{r warning = FALSE, message = FALSE}
data1 <- NSLC_6A %>%
  filter(`proteome_k3 (out of k4)` == 2) %>%
  mutate(OverallStaging =  factor(OverallStaging)) %>%
  mutate(P56856= as.numeric(P56856)) %>%
  mutate(Q8N3Y7= as.numeric(Q8N3Y7)) %>%
  mutate(P14091= as.numeric(P14091)) %>%
  mutate(Q9HCY8= as.numeric(Q9HCY8)) %>%
  mutate(Q16625= as.numeric(Q16625)) 
p <- ggplot(rbind(data1, data1), aes(x = OverallStaging, y = P56856, color = Gender))
q <- ggplot(rbind(data1, data1), aes(x = OverallStaging, y = Q8N3Y7, color = Gender))
r <- ggplot(rbind(data1, data1), aes(x = OverallStaging, y = P14091, color = Gender))
s <- ggplot(rbind(data1, data1), aes(x = OverallStaging, y = Q9HCY8, color = Gender))
t <- ggplot(rbind(data1, data1), aes(x = OverallStaging, y = Q16625, color = Gender))
```


```{r warning = FALSE, message = FALSE}
graph6 <- p +
  geom_violin(scale = "width", alpha = 0.3, adjust = 0.5, aes(fill = Gender)) +
  geom_boxplot(position = position_dodge(0.9), width = 0.2) +
  geom_point(position = position_jitterdodge(jitter.width = 0.3, jitter.height = 0, seed = 12), alpha = 0.9) +
  stat_summary(fun="mean", geom="point",shape = 23, size=5, aes(fill = Gender))  +
  labs(title = "Proteome subtype 2 - Gene CLDN18",
       x = "Stage",
       y = "Gene CLDN18") +
  theme_minimal_grid(12) 
```


```{r warning = FALSE, message = FALSE}
graph7 <- q +
  geom_violin(scale = "width", alpha = 0.3, adjust = 0.5, aes(fill = Gender)) +
  geom_boxplot(position = position_dodge(0.9), width = 0.2) +
  geom_point(position = position_jitterdodge(jitter.width = 0.3, jitter.height = 0, seed = 12), alpha = 0.9) +
  stat_summary(fun="mean", geom="point",shape = 23, size=5, aes(fill = Gender)) +
  labs(title = "Proteome subtype 2 - Gene SDR16C5",
       x = "Stage",
       y = "Gene SDR16C5") +
  theme_minimal_grid(12) 
```


```{r warning = FALSE, message = FALSE}
graph8 <- r +
  geom_violin(scale = "width", alpha = 0.3, adjust = 0.5, aes(fill = Gender)) +
  geom_boxplot(position = position_dodge(0.9), width = 0.2) +
  geom_point(position = position_jitterdodge(jitter.width = 0.3, jitter.height = 0, seed = 12), alpha = 0.9) +
  stat_summary(fun="mean", geom="point",shape = 23, size=5, aes(fill = Gender)) +
  labs(title = "Proteome subtype 2 - Gene CTSE",
       x = "Stage",
       y = "Gene CTSE") +
  theme_minimal_grid(12)  
```


```{r warning = FALSE, message = FALSE}
graph9 <- s +
  geom_violin(scale = "width", alpha = 0.3, adjust = 0.5, aes(fill = Gender)) +
  geom_boxplot(position = position_dodge(0.9), width = 0.2) +
  geom_point(position = position_jitterdodge(jitter.width = 0.3, jitter.height = 0, seed = 12), alpha = 0.9) +
  stat_summary(fun="mean", geom="point",shape = 23, size=5, aes(fill = Gender)) +
  labs(title = "Proteome subtype 2 - Gene S100A14",
       x = "Stage",
       y = "Gene S100A14") +
  theme_minimal_grid(12)
```


```{r warning = FALSE, message = FALSE}
graph10 <- t+
  geom_violin(scale = "width", alpha = 0.3, adjust = 0.5, aes(fill = Gender)) +
  geom_boxplot(position = position_dodge(0.9), width = 0.2) +
  geom_point(position = position_jitterdodge(jitter.width = 0.3, jitter.height = 0, seed = 12), alpha = 0.9) +
  stat_summary(fun="mean", geom="point",shape = 23, size=5, aes(fill = Gender)) +
  labs(title = "Proteome subtype 2 - Gene OCLN",
       x = "Stage",
       y = "Gene OCLN") +
  theme_minimal_grid(12)
```


```{r warning = FALSE, message = FALSE}
graph11 <- NSLC_6A %>%
  filter(`proteome_k3 (out of k4)` == 3) %>%
  mutate(OverallStaging =  factor(OverallStaging)) %>%
  mutate(P15088= as.numeric(P15088)) %>%
  ggplot(aes(x = OverallStaging, y = P15088, color = Gender)) +
  geom_violin(scale = "width", alpha = 0.3, adjust = 0.5, aes(fill = Gender)) +
  geom_boxplot(position = position_dodge(0.9), width = 0.2) +
  geom_point(position = position_jitterdodge(jitter.width = 0.3, jitter.height = 0, seed = 12), alpha = 0.9) +
  stat_summary(fun="mean", geom="point",shape = 23, size=5, aes(fill = Gender)) +
  labs(title = "Proteome subtype 3 - Gene CPA3",
       x = "Stage",
       y = "Gene CPA3") +
  theme_minimal_grid(12) 
```


```{r warning = FALSE, message = FALSE}
graph12 <- NSLC_6A %>%
  filter(`proteome_k3 (out of k4)` == 3) %>%
  mutate(OverallStaging =  factor(OverallStaging)) %>%
  mutate(P23946= as.numeric(P23946)) %>%
  ggplot(aes(x = OverallStaging, y = P23946, color = Gender)) +
  geom_violin(scale = "width", alpha = 0.3, adjust = 0.5, aes(fill = Gender)) +
  geom_boxplot(position = position_dodge(0.9), width = 0.2) +
  geom_point(position = position_jitterdodge(jitter.width = 0.3, jitter.height = 0, seed = 12), alpha = 0.9) +
  stat_summary(fun="mean", geom="point",shape = 23, size=5, aes(fill = Gender)) +
  labs(title = "Proteome subtype 3 - Gene CMA1",
       x = "Stage",
       y = "Gene  CMA1") +
  theme_minimal_grid(12) 
```


```{r warning = FALSE, message = FALSE}
graph13 <- NSLC_6A %>%
  filter(`proteome_k3 (out of k4)` == 3) %>%
  mutate(OverallStaging =  factor(OverallStaging)) %>%
  mutate(P55083= as.numeric(P55083)) %>%
  ggplot(aes(x = OverallStaging, y = P55083, color = Gender)) +
  geom_violin(scale = "width", alpha = 0.3, adjust = 0.5, aes(fill = Gender)) +
  geom_boxplot(position = position_dodge(0.9), width = 0.2) +
  geom_point(position = position_jitterdodge(jitter.width = 0.3, jitter.height = 0, seed = 12), alpha = 0.9) +
  stat_summary(fun="mean", geom="point",shape = 23, size=5, aes(fill = Gender)) +
  labs(title = "Proteome subtype 3 - Gene MFAP4",
       x = "Stage",
       y = "Gene MFAP4") +
  theme_minimal_grid(12) 
```


```{r warning = FALSE, message = FALSE}
graph14 <- NSLC_6A %>%
  filter(`proteome_k3 (out of k4)` == 3) %>%
  mutate(OverallStaging =  factor(OverallStaging)) %>%
  mutate(P07451= as.numeric(P07451)) %>%
  ggplot(aes(x = OverallStaging, y = P07451, color = Gender)) +
  geom_violin(scale = "width", alpha = 0.3, adjust = 0.5, aes(fill = Gender)) +
  geom_boxplot(position = position_dodge(0.9), width = 0.2) +
  geom_point(position = position_jitterdodge(jitter.width = 0.3, jitter.height = 0, seed = 12), alpha = 0.9) +
  stat_summary(fun="mean", geom="point",shape = 23, size=5, aes(fill = Gender)) +
  labs(title = "Proteome subtype 3 - Gene CA3",
       x = "Stage",
       y = "Gene CA3") +
  theme_minimal_grid(12) 
```


```{r warning = FALSE, message = FALSE}
graph15 <- NSLC_6A %>%
  filter(`proteome_k3 (out of k4)` == 3) %>%
  mutate(OverallStaging =  factor(OverallStaging)) %>%
  mutate(O75635= as.numeric(O75635)) %>%
  ggplot(aes(x = OverallStaging, y = O75635, color = Gender)) +
  geom_violin(scale = "width", alpha = 0.3, adjust = 0.5, aes(fill = Gender)) +
  geom_boxplot(position = position_dodge(0.9), width = 0.2) +
  geom_point(position = position_jitterdodge(jitter.width = 0.3, jitter.height = 0, seed = 12), alpha = 0.9) +
  stat_summary(fun="mean", geom="point",shape = 23, size=5, aes(fill = Gender)) +
  labs(title = "Proteome subtype 3 - Gene SERPINB7",
       x = "Stage",
       y = "Gene SERPINB7") +
  theme_minimal_grid(12) 
```

Now print plots.

```{r warning = FALSE, message = FALSE, fig.width = 33, fig.height = 45}
plot_grid(graph1, graph6, graph11, graph2, graph7, graph12, graph3, graph8, graph13, graph4, graph9, graph14, graph5, graph10, graph15, ncol = 3, labels = c('1A', '2A', '3A', '1B', '2B', '3B', '1C', '2C', '3C', '1D', '2D', '3D', '1E', '2E', '3E'))
```
It is found out that male's expression in selected genes is higher than female's expression especially in proteome subtype 1 and 3 in general. However, it was difficult to analyse the proteome subtype 2 due to the lack of samples. 


Now, I compared the log2T/N distribution of 'mRNA expression' and 'proteome expression' by stage per genders from the selected genes which showed the high average value from the each proteome subtype.

To find out this conveniently, we will extract the genes selected above and manipulate the data.
```{r warning = FALSE, message = FALSE}
NSLC_1A <- NSLC_1A[,-c(2,4,6,8,9)]

#Table NSLC_1D - mRNA expression results.
NSLC_1D <- NSLC_1D[,-c(2,3,32)]
NSLC_1D <- NSLC_1D %>% 
  filter(gene %in% c("CPA3", "CMA1", "MFAP4", "CA3", "SERPINB7","SFRP2", "CILP", "VSTM2L", "THBS2", "COL11A1", "CLDN18", "SDR16C5", "CTSE",  "S100A14", "OCLN"))
NSLC_1D <- pivot_longer(data = NSLC_1D, cols = -gene, names_to = 'ID', values_to = 'log2T/N' )
NSLC_1D <- merge(
  x=NSLC_1D,
  y=NSLC_1A,
  by='ID',
  all.x = TRUE
)

#Table NSLC_1E - Proteomic expression results.
NSLC_1E <- NSLC_1E %>% 
  filter(Gene %in% c("CPA3", "CMA1", "MFAP4", "CA3", "SERPINB7","SFRP2", "CILP", "VSTM2L", "THBS2", "COL11A1", "CLDN18", "SDR16C5", "CTSE",  "S100A14", "OCLN"))
NSLC_1E <- NSLC_1E[,-c(1,3)]
NSLC_1E <- pivot_longer(data = NSLC_1E, cols = -Gene, names_to = 'ID', values_to = 'log2T/N' )
NSLC_1E <- merge(
  x=NSLC_1E,
  y=NSLC_1A,
  by='ID',
  all.x = TRUE
)
```

Now lets' make plots what I want. 

```{r warning = FALSE, message = FALSE}
graph16 <- NSLC_1D %>%
  filter(gene %in% c("SFRP2", "CILP", "VSTM2L", "THBS2", "COL11A1")) %>%
  mutate(`Smoking Status` = ifelse(`Smoking Status` == "Nonsmoke", "non-smoker", "smoker")) %>%
  mutate(Stage = case_when(
    Stage == "IA" ~ "IA",
    Stage == "IB" ~ "IB",
    TRUE ~ ">=III"
  )) %>%
  ggplot(aes(x = `log2T/N`, y = gene, color = Gender)) +
  geom_density_ridges(aes(fill = Gender), alpha = 0.5, stat = "binline") +
  xlim(-5,10) +
  scale_fill_manual(values = c("#D55E0050", "#0072B250"), labels = c("Female", "Male")) +
  scale_color_manual(values = c("#D55E00", "#0072B2"), guide = "none") +
  facet_grid(Stage ~ `Smoking Status`) +
  labs(title = "Proteome subtype 1 - mRNA expression",
       x = "log2T/N",
       y = "Gene") +
  theme_minimal_grid(12)
```


```{r warning = FALSE, message = FALSE}
graph17 <- NSLC_1D %>%
  filter(gene %in% c("CLDN18", "SDR16C5", "CTSE",  "S100A14", "OCLN")) %>%
  mutate(`Smoking Status` = ifelse(`Smoking Status` == "Nonsmoke", "non-smoker", "smoker")) %>%
  mutate(Stage = case_when(
    Stage == "IA" ~ "IA",
    Stage == "IB" ~ "IB",
    TRUE ~ ">=III"
  )) %>%
  ggplot(aes(x = `log2T/N`, y = gene, color = Gender)) +
  geom_density_ridges(aes(fill = Gender), alpha = 0.5, stat = "binline") +
  xlim(-5,10) +
  scale_fill_manual(values = c("#D55E0050", "#0072B250"), labels = c("Female", "Male")) +
  scale_color_manual(values = c("#D55E00", "#0072B2"), guide = "none") +
  facet_grid(Stage ~ `Smoking Status`) +
  labs(title = "Proteome subtype 2 - mRNA expression",
       x = "log2T/N",
       y = "Gene") +
  theme_minimal_grid(12)
```


```{r warning = FALSE, message = FALSE}
graph18 <- NSLC_1D %>%
  filter(gene %in% c("CPA3", "CMA1", "MFAP4", "CA3", "SERPINB7")) %>%
  mutate(`Smoking Status` = ifelse(`Smoking Status` == "Nonsmoke", "non-smoker", "smoker")) %>%
  mutate(Stage = case_when(
    Stage == "IA" ~ "IA",
    Stage == "IB" ~ "IB",
    TRUE ~ ">=III"
  )) %>%
  ggplot(aes(x = `log2T/N`, y = gene, color = Gender)) +
  geom_density_ridges(aes(fill = Gender), alpha = 0.5, stat = "binline") +
  xlim(-5,10) +
  scale_fill_manual(values = c("#D55E0050", "#0072B250"), labels = c("Female", "Male")) +
  scale_color_manual(values = c("#D55E00", "#0072B2"), guide = "none") +
  facet_grid(Stage ~ `Smoking Status`) +
  labs(title = "Proteome subtype 3 - mRNA expression",
       x = "log2T/N",
       y = "Gene") +
  theme_minimal_grid(12)
```


```{r warning = FALSE, message = FALSE}
graph19 <- NSLC_1E %>%
  filter(Gene %in% c("SFRP2", "CILP", "VSTM2L", "THBS2", "COL11A1")) %>%
  mutate(`Smoking Status` = ifelse(`Smoking Status` == "Nonsmoke", "non-smoker", "smoker")) %>%
  mutate(Stage = case_when(
    Stage == "IA" ~ "IA",
    Stage == "IB" ~ "IB",
    TRUE ~ ">=III"
  )) %>%
  ggplot(aes(x = `log2T/N`, y = Gene, color = Gender)) +
  geom_density_ridges(aes(fill = Gender), alpha = 0.5, stat = "binline") +
  xlim(-3.5,5) +
  scale_fill_manual(values = c("#D55E0050", "#0072B250"), labels = c("Female", "Male")) +
  scale_color_manual(values = c("#D55E00", "#0072B2"), guide = "none") +
  facet_grid(Stage ~ `Smoking Status`) +
  labs(title = "Proteome subtype 1 - Proteomic expression",
       x = "log2T/N",
       y = "Gene") +
  theme_minimal_grid(12)
```


```{r warning = FALSE, message = FALSE}
graph20 <- NSLC_1E %>%
  filter(Gene %in% c("CLDN18", "SDR16C5", "CTSE",  "S100A14", "OCLN")) %>%
  mutate(`Smoking Status` = ifelse(`Smoking Status` == "Nonsmoke", "non-smoker", "smoker")) %>%
  mutate(Stage = case_when(
    Stage == "IA" ~ "IA",
    Stage == "IB" ~ "IB",
    TRUE ~ ">=III"
  )) %>%
  ggplot(aes(x = `log2T/N`, y = Gene, color = Gender)) +
  geom_density_ridges(aes(fill = Gender), alpha = 0.5, stat = "binline") +
  xlim(-3.5,5) +
  scale_fill_manual(values = c("#D55E0050", "#0072B250"), labels = c("Female", "Male")) +
  scale_color_manual(values = c("#D55E00", "#0072B2"), guide = "none") +
  facet_grid(Stage ~ `Smoking Status`) +
  labs(title = "Proteome subtype 2 - Proteomic expression",
       x = "log2T/N",
       y = "Gene") +
  theme_minimal_grid(12)
```


```{r warning = FALSE, message = FALSE}
graph21 <- NSLC_1E %>%
  filter(Gene %in% c("CPA3", "CMA1", "MFAP4", "CA3", "SERPINB7")) %>%
  mutate(`Smoking Status` = ifelse(`Smoking Status` == "Nonsmoke", "non-smoker", "smoker")) %>%
  mutate(Stage = case_when(
    Stage == "IA" ~ "IA",
    Stage == "IB" ~ "IB",
    TRUE ~ ">=III"
  )) %>%
  ggplot(aes(x = `log2T/N`, y = Gene, color = Gender)) +
  geom_density_ridges(aes(fill = Gender), alpha = 0.5, stat = "binline") +
  xlim(-3.5,5) +
  scale_fill_manual(values = c("#D55E0050", "#0072B250"), labels = c("Female", "Male")) +
  scale_color_manual(values = c("#D55E00", "#0072B2"), guide = "none") +
  facet_grid(Stage ~ `Smoking Status`) +
  labs(title = "Proteome subtype 3 - Proteomic expression",
       x = "log2T/N",
       y = "Gene") +
  theme_minimal_grid(12)
```

Now print plots.

```{r warning = FALSE, message = FALSE, fig.width = 12, fig.height = 27}
plot_grid(graph16, graph19, graph17, graph20, graph18, graph21, ncol = 2, labels = c('1A', '2A', '1B', '2B', '1C', '2C'))
```

First, It is found out that the value of proteome log2T/N is relatively bigger in general when the selected genes value were analyzed between the transcriptome log2T/N and proteome log2T/N. 
In general, We can also know that non-smoker’s  log2T/N difference between genders is not significant. However, when the log2T/N difference between smoker and non-smoker is compared, it is found out that the smoker’s log2T/N is slightly higher. 

This data analysis would be more reliable if the number of the patients were much larger. The number of patients in three proteome subtypes was not evenly distributed. When the data was analyzed per each proteome subtype, the number of subtype 2 was relatively less than others.






